src/aws/lambda-convert/index.js

```diff
@@ -67,6 +79,18 @@ exports.handler = async (event, context, callback) => {
       );
       break;
 
+    case "sharedlivemedia":
+      try {
+        const nbPages = await convertSharedLiveMedia(objectKey, sourceBucket);
+        await updateState(objectKey, READY, nbPages);
```

Comment:
In our case the extension is also expected when calling `updateState`. We will need in django. For now it's only `pdf` but it's subject to evolve and can be done now.

Reply:
The source, sorry my comment was not precise enough